cli/README.md

```diff
@@ -23,21 +23,31 @@ Options:
   -h, --help       display help for command
 
 Commands:
-  generate [path]  generate a new Grouparoo project
-  upgrade [path]   upgrade an existing Grouparoo project
-  help [command]   display help for command
+  config [options]                 Interactively configure Grouparoo.
```

Comment:
Hmm, this command isn't part of the initial set of commands available, and only shows up when within a Grouparoo project (as the next "Commands" section indicates". Could having `config` here cause confusion?

Reply:
Good point.  On that same note, `install`/`uninstall` probably don't belong here as those would only be in projects as well.

Reply:
Agreed, I'll take that out.  I'm somewhat surprised that `install` and `uninstall` are in there before `init`.  Axing `config` here.